src/targets/betafpvf411rx/target.h

```diff
@@ -23,6 +23,30 @@
 #define BUZZER_PIN PIN_A14
 //#define BUZZER_INVERT
 
+//--------------------------------------------------------------
+// RGB LEDs  --- Not currently working --
+//--------------------------------------------------------------
+#define RGB_LED_NUMBER 0
+//--------------------------------------------------------------
+// DMA => TIM3-CC4 => DMA1, Channel5, Stream2 (See drv_dma.c)
+//--------------------------------------------------------------
+#define RGB_LED_DMA        DMA_DEVICE_TIM3_CH4
+//--------------------------------------------------------------
+// pin / port for the RGB led
+//--------------------------------------------------------------
+#define RGB_PIN LL_GPIO_PIN_1
+#define RGB_PORT GPIOB
+#define RGB_PIN_CLOCK LL_AHB1_GRP1_PERIPH_GPIOB
+//--------------------------------------------------------------
+// Timer for the data signal => TIM2
+//--------------------------------------------------------------
+#define  RGB_TIM_CLOCK     LL_APB1_GRP1_PERIPH_TIM3
+#define  RGB_TIMER         TIM3
+#define  RGB_TIM_CHAN      LL_TIM_CHANNEL_CH4
+#define  RGB_TIM_CCR       CCR4
+#define  RGB_TIM_AF        GPIO_AF2_TIM3
```

Comment:
we _probably_ wanna merge this somehow with the motor (pwm) configuration. see motor_pins.h
likely by eliminating the timer defs in motor_pins.h and adding a lookup table to drv_timer.c

Reply:
Yes, definitely agree here. Would be nice to just need to specify the pin, and let the code look up what timer AF that pin has, and ideally the dma port/stream/channel that services that timer(channel).

Reply:
Would this block the PR? Or can that be a project for a later time? I'm thinking this is not an easy job (I'm having trouble referencing TIMx objects in my code, so not sure how i can return that as a response from a lookup function). I do think this could be done better though, like you said... It should be possible to get those values by looking it up from the Port/Pin index specified in RGB_PIN